finish/README.txt

```diff
@@ -25,50 +23,42 @@ BankService.java file also contains code that automatically fails the first 2
 requests that are made to the checkBalance service so as to demonstrate the states 
 of a circuit breaker. 
 
-The @Fallback annotation and the fallback method are in a separate file, 
-BankServiceWithFallback.java. The fallback method is invoked whenever a request 
-to the service fails or when the circuit is open. When the fallback is running 
-the method immediately returns the following message: 
-	"The last known balance of your account is $10,293". 
+The @Fallback annotation and the fallback method are demonstrated in the same source 
+file. The fallback method is invoked whenever a request to the service fails or when 
+the circuit is open. When the circuit is opened, the request is not allowed to go 
+through and the fallback is called immediately. 
 
-To access the sample application with the checkBalance service, visit the 
-following URL from your browser: 
+To access the sample application with the checkBalance service that includes
+a fallback method, visit the following URL from your browser: 
       http://localhost:9080/circuitBreakerSample/checkBalance 
 The checkBalance service simulates access failures in order to demostrate the 
 circuit breaker in action. The first 2 attempts to access the service will fail. 
-Therefore, after the initial invocation of the checkBalance service the output 
+The fallback is called when the first request fails. The output from the fallback 
```

Comment:
I think we should still start this sentence with 'Therefore'......"**Therefore**, the fallback is called **after** the first request fails."

Reply:
Thank you, fixed.